packages/jest-runtime/src/index.js

```diff
@@ -515,20 +561,27 @@ class Runtime {
     );
 
     localModule.paths = this._resolver.getModulePaths(dirname);
+
     Object.defineProperty(localModule, 'require', {
       value: this._createRequireImplementation(filename, options),
     });
 
+    const fileSource = isNativeModule
+      ? // $FlowFixMe: process.binding exists.
+        process.binding('natives')[filename]
```

Comment:
is this call fast or slow?

Reply:
_Really_ fast. The following code:

```javascript
console.time('binding');
for (let i = 0; i < 100000; i++) {
  global.fsSourceCode = process.binding('natives').fs;
}
console.timeEnd('binding');
```

Reports an average of `16ms`.